x-pack/plugins/security/server/role_mappings/get_enabled_role_mappings_features.test.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { elasticsearchServiceMock, loggingServiceMock } from 'src/core/server/mocks';
+import { getEnabledRoleMappingsFeatures } from './get_enabled_role_mappings_features';
+
+describe('getEnabledRoleMappingsFeatures', () => {
+  // Most scenarios are covered in the `feature_check.test.ts` suite, so will not be repeated here
```

Comment:
Yeah I can certainly in-line this logic. It started out that way,  but then I split it out because the lengthy route handler looked a bit messy to be, but you’re right, it’s not adding any value standing on its own

Reply:
If you'd like to have it in a separate file, perhaps we could create a folder at `x-pack/plugins/security/server/routes/role_mapping/feature_check` and put this file within that folder?